types/framework/entity.d.ts

```diff
@@ -311,30 +314,15 @@ declare class Entity<D extends Entity.Data = Entity.Data> {
    * const created: Actor[] | null = await Actor.create(data, {temporary: true}); // Not saved to the database
    * ```
    */
-  static create<T extends Entity>(
-    this: ConstructorOf<T>,
-    data: DeepPartial<T['data']>,
-    options?: Entity.CreateOptions
-  ): Promise<T | null>;
-  static create<T extends Entity, D extends ReadonlyArray<DeepPartial<T['data']>>>(
-    this: ConstructorOf<T>,
-    data: D,
-    options?: Entity.CreateOptions
-  ): D extends never[]
-    ? Promise<[]>
-    : IsReadonlyArrayWithKey<D, 0> extends true
-    ? IsReadonlyArrayWithKey<D, 1> extends true
-      ? Promise<T[] | null>
-      : Promise<T | null>
-    : Promise<T | T[] | null>;
+  static create: Entity.CreateFunction<Entity>;
 
   /**
    * Handle a SocketResponse from the server when one or multiple Entities are created
    * @param request - The initial request
    * @param result  - An Array of created Entity data
    * @param userId  - The id of the requesting User
    */
-  protected static _handleCreate({ request, result, userId }: any): Entity[];
+  protected static _handleCreate<T extends Entity>(this: ConstructorOf<T>, { request, result, userId }: any): T[];
```

Comment:
I don't know if this needs to be generic. It's called by the foundry socket response, not user code.

Reply:
In theory, user code could call it. Otherwise, what's the point of giving a concrete return type at all. If we expect the user to never call this, we could just have it return `unknown`. And it doesn't hurt, does it?

Reply:
Also the user could override this, in which case he also need to know the correct return type.